### ISSUE TITLE
[macOS] Swar manager stops every time it start a new plot 

### ISSUE THREAD

**Issue by GameOSX** [swar/Swar-Chia-Plot-Manager#867]
I've got this problem - Swar Manager v.0.1.0 stops running every time it start a new plot, and I have to start it manually again and again.
In debug log it says:

````
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/psutil/_psosx.py", line 344, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/psutil/_psosx.py", line 562, in nice_set
    return cext_posix.setpriority(self.pid, value)
PermissionError: [Errno 13] Permission denied

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/aaa/Plot-Manager-Test/Swar-Chia-Plot-Manager/stateless-manager.py", line 94, in <module>
    jobs, running_work, next_job_work, next_log_check = monitor_jobs_to_start(
  File "/Users/aaa/Plot-Manager-Test/Swar-Chia-Plot-Manager/plotmanager/library/utilities/jobs.py", line 258, in monitor_jobs_to_start
    job, work = start_work(
  File "/Users/aaa/Plot-Manager-Test/Swar-Chia-Plot-Manager/plotmanager/library/utilities/jobs.py", line 327, in start_work
    psutil.Process(pid).nice(nice_val)
  File "/usr/local/lib/python3.9/site-packages/psutil/__init__.py", line 724, in nice
    self._proc.nice_set(value)
  File "/usr/local/lib/python3.9/site-packages/psutil/_psosx.py", line 351, in wrapper
    raise AccessDenied(self.pid, self._name)
psutil.AccessDenied: psutil.AccessDenied (pid=23768)
````

On previous version there wasn't such problem.

**Comment by wesray** [comment]
looks like you have a permission error.   Since it says permission denied.

run `sudo` in front of everything  for admin priviliges for now.

**Comment by GameOSX** [comment]
already tried - did not helped :/

**Comment by WasHupDa** [comment]
Hey guys,

I experienced the same problem here with Ubuntu 20.04.
I was completely clearing the debug log a few minutes before the new plot should start.
The plot was created correctly, but the manager stopped. The debug log is completely empty (0 Bytes).

I'm plotting 3 parallel plots on a 2TB SSD, with 32 GB RAM.
Stagger time is 3 hours (Phase 1 average time is about 2h 55m)


Anyone has an idea, why this is happening?

Best regards,
Peter

**Comment by phillipwonder** [comment]
I've used polt manager for many days.  I am runing on windows 10 , but tonight I‘ve got the same problem .   Miner status : sotpped .   And when tried to restart it .  It will show STOP in few minutes .  Anyone has the solution ？ Two machine has the same problem.

**Comment by Woolear** [comment]
Same for me. Linux

**Comment by ristraman** [comment]
I had a similar experience last night on Windows 10. I was getting some less than optimum plot times, stopped the manager on two machines, changed some parameters (increased threads, reduced max parallel plots) and started the manager again. I launched the viewer and when the still running plots appeared. When it was time for a new plot to launch, the status of the manager changed to stopped. I exited the viewer and restarted the manager, back to the viewer same thing happens (both machines). Went back to launching individual PS windows manually for now :-(
